index.html

```diff
@@ -15,7 +15,7 @@
     <meta name="generator" content="Prairie Creek Game">
     <meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
 
-    <title>My App</title>
+    <title>Spirits at Prairie Creek Park</title>
     <meta name="description" content="My App description">
```

Comment:
Whoops. Must have overlooked it. I'm at work now so I'll just have to fix this when I get done tonight